R/leaflet.R

```diff
@@ -19,15 +19,24 @@
 #' @param width the width of the map
 #' @param height the height of the map
 #' @param padding the padding of the map
+#' @param mapOptions the map options
 #' @return A HTML widget object, on which we can add graphics layers using
 #'   \code{\%>\%} (see examples).
 #' @example inst/examples/leaflet.R
 #' @export
-leaflet = function(data = NULL, width = NULL, height = NULL, padding = 0) {
+leaflet = function(data = NULL, width = NULL, height = NULL,
+                   padding = 0, mapOptions = list()) {
```

Comment:
I'd like you to rename the `mapOptions` argument as `options` (and make its default `mapOptions()`), and add a `mapOptions` function that takes crs/minzoom/maxzoom/maxbounds as arguments (center and zoom are covered by setView--or if you prefer we could have center/zoom in both places). This style is what we've used for the other functions and lets us provide autocomplete help.